cyences_app_for_splunk/appserver/static/notable_event_editor.js

```diff
@@ -406,10 +406,10 @@ require([
                     } else {
                         $td.html('<input type="checkbox" class="notable_event_selector" id="notable_event_selector" name="notable_event_selector" value="' + cell.value + '"></input>');
                     }
-                    $td.on("click", function (e) {
-                        e.stopPropagation();
-                        $td.trigger("cyences_notable_event_handlers", { "action": cell.field });
-                    });
+                    $td.children("input").on("click", function (e) {
```

Comment:
This will limit where use can click only on checkbox not on full cell.

Reply:
yes, it will limit only checkbox to clickable.